### PR TITLE
[CTCP-4746] updated CYA page with new prod details

### PR DESCRIPTION
**Pull request by prashanthmamidi**
As content is not ready yet - added the draft content for now

### CODE REVIEW

**Comment by prashanthmamidi** [comment]
image

**Review by Reece-Carruthers — DISMISSED** [review]

File: app/utils/answersHelpers/CheckYourAnswersHelper.scala [hunk 1]
```diff
@@ -50,6 +50,22 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messag
...
+    call = Some(controllers.routes.NewAuthYesNoController.onPageLoad(arrivalId, CheckMode))
+  )
+
+  def otherThingsToReport: Option[SummaryListRow] = getAnswerAndBuildRow[String](
+    page = OtherThingsToReportPage,
+    formatAnswer = formatAsText,
+    prefix = "checkYourAnswers.rowHeadings.otherThingsToReport",
+    id = Some("change-other-things-to-report"),
+    call = Some(controllers.routes.OtherThingsToReportController.onPageLoad(arrivalId, CheckMode))
+  )
```
Comment (jackheslop96) [inline app/utils/answersHelpers/CheckYourAnswersHelper.scala:67]: I believe this has already been implemented in the `report` method?

File: app/viewModels/CheckYourAnswersViewModel.scala [hunk 2]
```diff
@@ -42,7 +42,9 @@ object CheckYourAnswersViewModel {
           helper.unloadingType,
           helper.goodsUnloadedDate,
           helper.canSealsBeRead,
-          helper.anySealsBroken
+          helper.anySealsBroken,
+          helper.anyNewProcedure,
+          helper.otherThingsToReport
```
Comment (jackheslop96) [inline app/viewModels/CheckYourAnswersViewModel.scala:47]: This is already implemented in `commentsSection`

**Comment by prashanthmamidi** [comment]
image

**Pull request closed without merging**